src/main/resources/avro/reads.avdl

```diff
@@ -177,7 +177,38 @@ record LinearAlignment {
 }
 
 /**
-Each read alignment describes a linear alignment with additional information
+A string-to-reference-graph alignment can be represented by one CIGAR string and
+one `Path` through multiple `Reference`s, against which the CIGAR string is
+interpreted.
+
+Note that `Path`s are restricted to following reference adjacencies.
+If you want to align your read following adjacencies that are
+not present in the `ReferenceSet` to which you are aligning, you have a
+"chimeric" alignment, and should use multiple `ReadAlignment`s and the
+supplementaryAlignment flag instead of a single `GraphAlignment`.
+
+Some especially large deletions could be represented just as well as a large
+deletion in the CIGAR string, or as a chimeric alignment.
+*/
+record GraphAlignment {
+  /** The `Path` against which the read is aligned */
+  Path path;
+
+  /**
+  The mapping quality of this alignment. Represents how likely
+  the read maps to this position as opposed to other locations.
```

Comment:
It's supposed to be the same as for a linear alignment: the phred-scale probability that the mapping is wrong. However, we do need to work out what exactly we mean by "wrong" in both cases.